src/index.html

```diff
@@ -5,8 +5,341 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="page-top">
+    <header class="header">
+      <div class="logo">
+        <a href="#page-top">
+          <img src="./images/logo.png" alt="moyo logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#section-1" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-2" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-3" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#section-4"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-4" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-4" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-4" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#section-4" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="catalog-wrapper">
+        <div class="product-card-wrapper" data-qa="card">
+          <img
+            class="product-picture"
+            src="./images/imac.jpeg"
+            alt="product picture"
+          >
+          <div class="product-name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+          <div class="product-code">
+            Код товара: 195434
+          </div>
+          <div class="product-rating">
+            <div class="product-rating__stars">
```

Comment:
```suggestion
            <div class="product-rating__stars stars--4">
```
Тут еще должен быть такой модификатор, отвечающий за количество активных звезд